core/sail/federation/src/main/java/org/eclipse/rdf4j/sail/federation/Federation.java

```diff
@@ -94,6 +100,18 @@ public void addMember(Repository member) {
 		members.add(member);
 	}
 
+	protected List<Repository> getMembers() {
+		return Collections.unmodifiableList(members);
+	}
+
+	public void setBloomFilter(Repository member, RepositoryBloomFilter filter) {
```

Comment:
Should this method support the removal case? I can't remember if HashMap.put(key, null) is supported, but in general the Map.put contract says that it can throw an NPE if value is null.